### PR TITLE
feat: add tokens from defillama pools api

### PR DESCRIPTION
**Pull request by o-az**
#36 

I could not find a way that will reliably give me token info from address so I just copied this code from another codebase I wrote while back. It simply batches JSON RPC requests and checks name, symbol, and decimals for each batch param:

https://github.com/llamafolio/llamafolio-tokens/blob/010822d4496c400069161ec642db184ea4f8a2ae/scripts/token.ts#L98-L171

### CODE REVIEW

File: fantom/tokenlist.json [hunk 1]
```diff
@@ -1930,5 +1930,264 @@
     "symbol": "ZOO",
     "decimals": 0,
     "stable": false
+  },
+  {
+    "address": "0x07e6332dd090d287d3489245038daf987955dcfb",
```
Comment (0xsign) [inline fantom/tokenlist.json:1935]: Geist protocol

File: fantom/tokenlist.json [hunk 2]
```diff
@@ -1930,5 +1930,264 @@
...
+  {
+    "address": "0x524cabe5b2f66cbd6f6b08def086f18f8dde033a",
+    "name": "Biconomy Token",
+    "symbol": "BICO",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x58e57ca18b7a47112b877e31929798cd3d703b0f",
+    "name": "Curve.fi USD-BTC-ETH",
```
Comment (0xsign) [inline fantom/tokenlist.json:2013]: Curve protocol

**Review by 0xsign — COMMENTED** [review]
There are a couple of tokens from protocols, could you remove them by looking at the current addresses we have stored ?

**Comment by o-az** [comment]
> There are a couple of tokens from protocols, could you remove them by looking at the current addresses we have stored ?

✅

File: fantom/tokenlist.json [hunk 3]
```diff
@@ -1930,5 +1846,201 @@
...
+  },
+  {
+    "address": "0xe3e0eba90185e7292b9ca92c7409cdef504f9d07",
+    "name": "FUCK",
+    "symbol": "FUCK",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0xe578c856933d8e1082740bf7661e379aa2a30b26",
```
Comment (0xsign) [inline fantom/tokenlist.json:2012]: remove gUSDC (Geist Finance)

File: polygon/tokenlist.json [hunk 4]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x60d01ec2d5e98ac51c8b4cf84dfcce98d527c747",
+    "name": "izumi Token",
+    "symbol": "iZi",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x60d55f02a771d515e077c9c2403a1ef324885cec",
```
Comment (0xsign) [inline polygon/tokenlist.json:7479]: remove amUSDT (AAVE)

File: polygon/tokenlist.json [hunk 5]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x60138efceb1f7d8e10c8cb89bb61bbdbeebb4ffc",
+    "name": "RICE",
+    "symbol": "RICE",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x604229c960e5cacf2aaeac8be68ac07ba9df81c3",
```
Comment (0xsign) [inline polygon/tokenlist.json:7465]: remove UNI-V2 (Uniswap LP token)

File: polygon/tokenlist.json [hunk 6]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x5bf9496f0cd1f71c3353c3766f510772d7553b4f",
+    "name": "Sensorium",
+    "symbol": "SENSO",
+    "decimals": 0,
+    "stable": false
+  },
+  {
+    "address": "0x5c2ed810328349100a66b82b78a1791b101c9d61",
```
Comment (0xsign) [inline polygon/tokenlist.json:7409]: remove amWBTC (AAVE)

File: polygon/tokenlist.json [hunk 7]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x283366bb42ef49a994913baf22263c6562e588a4",
+    "name": "PENT",
+    "symbol": "PENT",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x28424507fefb6f7f8e9d3860f56504e4e5f5f390",
```
Comment (0xsign) [inline polygon/tokenlist.json:6835]: remove amWETH (AAVE)

File: polygon/tokenlist.json [hunk 8]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x27e56fc93a7c9f7ad220c1206ef03ed303bd648b",
+    "name": "Mira Token",
+    "symbol": "MIRA",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x27f8d03b3a2196956ed754badc28d73be8830a6e",
```
Comment (0xsign) [inline polygon/tokenlist.json:6814]: remove amDAI (AAVE)

File: polygon/tokenlist.json [hunk 9]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x200c234721b5e549c3693ccc93cf191f90dc2af9",
+    "name": "METAL",
+    "symbol": "METAL",
+    "decimals": 18,
+    "stable": false
+  },
+  {
+    "address": "0x20d3922b4a1a8560e1ac99fba4fade0c849e2142",
```
Comment (0xsign) [inline polygon/tokenlist.json:6723]: remove maWETH (AAVE)

File: polygon/tokenlist.json [hunk 10]
```diff
@@ -6340,5 +6319,2777 @@
...
+  },
+  {
+    "address": "0x19c60a251e525fa88cd6f3768416a8024e98fc19",
+    "name": "Wrapped amUSDT",
+    "symbol": "amUSDT",
+    "decimals": 6,
+    "stable": true
+  },
+  {
+    "address": "0x1a13f4ca1d028320a707d99520abfefca3998b7f",
```
Comment (0xsign) [inline polygon/tokenlist.json:6611]: remove amUSDC (AAVE)

**Pull request closed without merging**